utils/BaseScript.sol

```diff
@@ -11,7 +11,7 @@ abstract contract BaseScript is DeployScript {
     Constants internal immutable constants = new Constants(vm);
     bool internal testing;
 
-    function deployer() public view returns (address) {
+    function _deployer() public view returns (address) {
```

Comment:
Oh wait, this was done to make it compile
But this is not what you want as places where they were getting this, they will now get the forge-deploy's deploy contract address

Reply:
I'll just remove it, we never use it anyway :)